### PR TITLE
Alerts API lab

### PR DESCRIPTION
**Pull request by jmartin4563**
note: this replaces #257, not sure why but git was unhappy with me pushing to that branch.

### CODE REVIEW

File: content/api/alerts.apib [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    {
+        "results": {
+            "id": "cbe0faf0-9051-11e8-ad04-490e82764b23",
+            "name": "Sparky's Monthly Alert",
+            "enabled": true,
+            "alert_metric": "monthly_sending_limit"
+        }
+    }
+
+### List all Alerts [GET /labs/alerts]
```
Comment (jgzamora) [inline content/api/alerts.apib]: Please move List to the end of the doc

File: content/api/alerts.apib [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    }
+
+
+### Updates an Alert [PUT /labs/alerts/{id}]
+Updates an existing alert
+
++ Data Structure
+    + name (string) - Display name of alert
+    + description (string) - Description of alert
+    + email_addresses (array) - List of email addresses to send alert notifications
```
Comment (jgzamora) [inline content/api/alerts.apib]: Add limit like in create

File: content/api/alerts.apib [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    + threshold (object) - Configuration for level and threshold at which an alert is generated
+        + error (object) - Error level alert configuration
+            + target (number) - Percentage value at which an alert will be generated
+            + comparator (enum) - Conditional comparison operator used by alerts service to compare actual value of `alert_metric` to the `target` value
+                + Default: `"gt"`
+    + enabled (boolean) - Whether or not the alert is active
+        + Default: true
```
Comment (jgzamora) [inline content/api/alerts.apib]: ❌ defaults in the update body

File: content/api/alerts.apib [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+            }
+        },
+        "enabled": true
+    }
+    ```
+
+### Create an Alert [POST /labs/alerts]
+Creates a new alert
+
++ Data Structure
```
Comment (avigoldman) [inline content/api/alerts.apib]: What is required here?

File: content/api/alerts.apib [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+                "target": 90,
+                "comparator": "gt"
+            }
+        },
+        "enabled": true
+    }
+
+
+### Updates an Alert [PUT /labs/alerts/{id}]
+Updates an existing alert. Updating email_addresses replaces the the existing array with the new array.
```
Comment (avigoldman) [inline content/api/alerts.apib]: backticks on `email_addresses`

File: content/api/alerts.apib (hunk 4 above)
Comment (jmartin4563) [reply content/api/alerts.apib]: whoopsie, updating. it's `name`, `email_addresses`, `alert_metric`, `threshold.error.target` :)

**Pull request merged**